graph/api/src/main/scala/org/apache/spark/graph/api/GraphElementDataset.scala

```diff
@@ -17,17 +17,17 @@
 
 package org.apache.spark.graph.api
 
+import org.apache.spark.annotation.Evolving
 import org.apache.spark.sql.{Dataset, Row}
 
 /**
- * A [[PropertyGraph]] is created from GraphElementFrames.
- *
- * A graph element is either a node or a relationship.
- * A GraphElementFrame wraps a Dataset and describes how it maps to graph elements.
+ * A [[PropertyGraph]] whose graph element is either a node or a relationship.
```

Comment:
It isn't exactly a `PropertyGraph`, but a building block towards that. I suggest rewording like this
```suggestion
 * A [[PropertyGraph]] component containing graph elements; either nodes or relationships.
```